### ISSUE TITLE
Feature Request: User Groups

### ISSUE THREAD

**Issue by alexlewer** [phpservermon/phpservermon#190]
It would be great to be able to define groups of users which would make adding new users to existing server monitors much easier.

**Comment by mooleshacat** [comment]
basically same as / duplicate:

https://github.com/phpservermon/phpservermon/issues/739
https://github.com/phpservermon/phpservermon/issues/746

**Comment by viharm** [comment]
> basically same as / duplicate:
> 
> #739 #746

No, this is not a duplicate. #739 refers to grouping the servers. #746 refers to grouping monitors by servers.

This is about grouping users.

I would love this feature too :-)
